db/migrate/20220705115226_create_users.rb

```diff
@@ -0,0 +1,8 @@
+class CreateUsers < ActiveRecord::Migration[7.0]
+  def change
+    create_table :users do |t|
+
+      t.timestamps
+    end
+  end
+end
```

Comment:
- [ ] An error occurs at the point when the Users table is being created while running your migrations. Please fix it 🙏. I have attached the error below 👇 
![image](https://user-images.githubusercontent.com/46839548/178037427-1745296d-3486-4dc2-b96b-14372c9e80c1.png)
The next reviewer will proceed to check other aspects of your project after you have fixed this issue

Reply:
HEY @codepantha ,sorry but i think this errors are on your side

Reply:
![screenshot 1](https://user-images.githubusercontent.com/81693905/178095050-9ed44a10-a729-4355-a663-04422f72512a.png)
Please see the attached images of my tests and migratio
![screenshot 2](https://user-images.githubusercontent.com/81693905/178095083-c9f10651-a3a4-4567-9d61-d762b72e6645.png)
n